app/views/groups/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= render "shared/nav" %>
+
+<div class="categories">
+  <% if @groups.empty? %>
+  <div class="get-started">
+    <h3 class="started">Get Started</h3>
+    <span class="div">Add your first budget by clicking on the button below</span>
+  </div>
+  <% else %>
+
+  <div class="categories-container">
+    <% @groups.each do |group| %>
+      <div class="category-card">
+        <div class="category-icon">
+          <%= image_tag group.icon, alt: 'icon', class: "icon" %>
+        </div>
+        <div class="card-text">
+        <%= link_to group_path(group.id.to_s), style: "text-decoration: none;" do %>
+          <p class="category-name"><%= group.name %></p>
+        <% end %>
+        <p>$<%= group.expenses.sum(:amount) %></p>
+        <%= link_to 'Learn more →', group_path(group.id.to_s), style: "text-decoration: none;", class: "learn" %>
+        <p class="user">by <%= current_user.name %></p>
+        </div>
+      </div>
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the items inside this card are displayed as flex so they can be on a row. Kindly compare your design with the design provided and fix this.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="301" alt="Screenshot 2023-03-29 at 00 13 59" src="https://user-images.githubusercontent.com/58692590/228389732-66d14795-5e05-4212-9bcb-fb030a79d1d7.png"> | <img width="400" alt="Screenshot 2023-03-29 at 00 14 20" src="https://user-images.githubusercontent.com/58692590/228389762-42532350-8426-4ea7-ae51-0522d4f06d99.png">

Reply:
Thank you @Whoistolu , I will update change the display